eoscostarica/guide.eoscostarica.io
# Feat/readme file templates

### GH Issue

Resolves #202  

### Steps to test
1. Open developer guidelines tag
1. Check README template tag exist
1. Check content is right

#### CheckList
